### PR TITLE
Added an outreach task for labyrinth, Fixed many typo(s)

### PR DESCRIPTION
**Pull request by 0x48piraj**
Things accomplished :

### CODE REVIEW

File: tasks/create-promotional-video-about-the-Labyrinth-game.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Labyrinth: Create a Promotional Video about the Labyrinth Game
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To promote the labyrinth, create a promotional video.
+
+REQUIREMENTS:
+
+- Explore [Video Creating](https://en.wikipedia.org/wiki/List_of_video_editing_software#Free_and_open-source)
+- Include the FOSSASIA logo. You find **FOSSASIA Artwork** on this repository: ***https://github.com/fossasia/fossasia-artwork***
+- Use existing [YouTube videos from the FOSSASIA Channel](https://youtube.com/fossasiaorg) and remix them
```
Comment (niccokunzmann) [inline tasks/create-promotional-video-about-the-Labyrinth-game.md]: Is this a requirement or a hint and optional?

File: tasks/create-promotional-video-about-the-Labyrinth-game.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- Link the video on the program website. Add it to ***https://github.com/fossasia/gci17.fossasia.org***
+- Tweet the video on Twitter, Weibo or other tweet like channel and include the video in the tweet itself. For example something like "Check out this video about #FOSSASIA http-youtube-link @hpdang @mariobehling @fossasia @catallman More at https://github.com/fossasia/labyrinth"
+- Post about your video on Facebook and other social media channels. Your post should include the link to the video.
+
+EXPECTED OUTCOME:
+
+- Video made using Open Source on YouTube using FOSSASIA Artwork/logo
+- Public screening
+- Link on FOSSASIA Code-In site
+- Tweet and post it on social media
```
Comment (niccokunzmann) [inline tasks/create-promotional-video-about-the-Labyrinth-game.md]: Should we add it as a link to the labyrinth website?

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
What do you think about my comments? Would you like to add them or is this intended?

**Comment by 0x48piraj** [comment]
Mmm.. ? Which comments @niccokunzmann ?

File: tasks/create-promotional-video-about-the-Labyrinth-game.md (hunk 2 above)
Comment (0x48piraj) [reply tasks/create-promotional-video-about-the-Labyrinth-game.md]: We can upload the video to youtube and then can share the links.

> Any suggestions ?

File: tasks/create-promotional-video-about-the-Labyrinth-game.md (hunk 1 above)
Comment (0x48piraj) [reply tasks/create-promotional-video-about-the-Labyrinth-game.md]: It's a optional step, but will much be appreciated :smile:

**Pull request merged**